nautobot/extras/views.py

```diff
@@ -2277,7 +2276,7 @@ def get_extra_context(self, request, instance):
         }
 
 
-class ObjectChangeLogView(generic.GenericView, UIComponentsMixin):
+class ObjectChangeLogView(generic.GenericView):
```

Comment:
Any functional impact of removing the mixin here and below? How does this intersect with #7808?